kyuubi-server/src/test/resources/log4j2-test.xml

```diff
@@ -21,6 +21,7 @@
 <Configuration status="WARN">
```

Comment:
we have bunches of `log4j2-test.xml`, can we sync them?

Reply:
I think it is not needed, we just need to enable it in kyuubi-server side.